localstack/testing/pytest/marker_report.py

```diff
@@ -0,0 +1,92 @@
+import dataclasses
+import json
+import os.path
+from typing import TYPE_CHECKING, List
+
+import pytest
+
+if TYPE_CHECKING:
+    from _pytest.config import Config, PytestPluginManager
+    from _pytest.config.argparsing import Parser
+
+
+@dataclasses.dataclass
+class MarkerReportEntry:
+    node_id: str
+    file_path: str
+    markers: list[str]
+
+
+@dataclasses.dataclass
+class MarkerReport:
+    prefix_filter: str
+    entries: list[MarkerReportEntry] = dataclasses.field(default_factory=list)
+    aggregated_report: dict[str, int] = dataclasses.field(default_factory=dict)
+
+    def create_aggregated_report(self):
+        for entry in self.entries:
+            for marker in entry.markers:
+                self.aggregated_report.setdefault(marker, 0)
+                self.aggregated_report[marker] += 1
+
+
+@pytest.hookimpl
+def pytest_addoption(parser: "Parser", pluginmanager: "PytestPluginManager"):
+    """
+    Standard usage. Will create a report for all markers under ./target/marker-report-<date>.json
```

Comment:
the file name is currently only `report.json` -> could you either change the docs, or adapt the actual name of the output report? 🙏